méli-mélo/th-empathy/style.css

```diff
@@ -0,0 +1,98 @@
+/* Thematic purple */
+.panel-empathy,
+hr.brdr-empathy {
```

Comment:
```suggestion
.hr-empathy {
```

Reply:
This section has been removed since it's currently not used in this thematic.